packages/govuk-frontend/tasks/watch.mjs

```diff
@@ -57,5 +57,12 @@ export const watch = (options) => gulp.parallel(
     gulp.watch([
       `${slash(options.srcPath)}/govuk/**/*.{md,njk}`
     ], templates(options))
+  ),
+
+  // Copy GOV.UK Frontend static assets
+  task.name('copy:assets watch', () =>
+    gulp.watch([
+      `${slash(options.srcPath)}/govuk/assets/`
```

Comment:
Mind if we use globs to target files, not directories?

Might be fine for assets, but we avoid a potential race condition where **Nodemon** (server-side) or **Browsersync** (client-side) might reload before the contents has finished copying

```suggestion
      `${slash(options.srcPath)}/govuk/assets/**`
```

Reply:
For context, it's largely because the macOS [File System Events (FSEvents) API](https://developer.apple.com/documentation/coreservices/file_system_events) used by the [`chokidar` watcher](https://github.com/paulmillr/chokidar) is replaced with basic "polling" on Windows

So when directories are removed by `clean()` we still need to wait for all the files to come back